### PR TITLE
Fix 3d tiles clipping planes sandcastle

### PR DESCRIPTION
**Pull request by lilleyse**
Error when calling `scene.primitives.removeAll` with a mix of primitives and entities. Same problem as https://github.com/AnalyticalGraphicsInc/cesium/pull/6367.

@hpinkos can you review?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@lilleyse, thanks for the pull request! Maintainers, we have a signed CLA from @lilleyse, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Apps/Sandcastle/gallery/3D Tiles Clipping Planes.html [hunk 1]
```diff
@@ -255,7 +255,6 @@
 
 function reset() {
     viewer.entities.removeAll();
-    viewer.scene.primitives.removeAll();
```
Comment (hpinkos) [inline Apps/Sandcastle/gallery/3D Tiles Clipping Planes.html]: I think you need to replace this with something like `viewer.scene.primitives.remove(tileset)`

**Comment by hpinkos** [comment]
`Camera.html` also calls `scene.primitives.removeAll` for removing the `DebugModelMatrixPrimitive` added in the `setReferenceFrame` section.  Change that to something like `scene.primitives.remove(referenceFramePrimitive)`

File: Apps/Sandcastle/gallery/3D Tiles Clipping Planes.html (hunk 1 above)
Comment (hpinkos) [reply Apps/Sandcastle/gallery/3D Tiles Clipping Planes.html]: Can you make that change to `Terrain Clipping Planes.html` too?

**Comment by lilleyse** [comment]
@hpinkos Updated

**Pull request merged**